tests/apollo/test_skvbc_commit_path.py

```diff
@@ -58,20 +87,19 @@ async def test_fast_path_is_default(self, bft_network, tracker, exchange_keys=Tr
 
         Finally the decorator verifies the KV execution.
         """
-
+        op_count = 5
         bft_network.start_all_replicas()
-        skvbc = kvbc.SimpleKVBCProtocol(bft_network,tracker)
-
-        # Initially all replicas are running on the fast path
-        await bft_network.wait_for_fast_path_to_be_prevalent(
-            run_ops=lambda: skvbc.run_concurrent_ops(num_ops=NUM_OPS, write_weight=1), threshold=NUM_OPS)
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network, tracker)
 
+        await bft_network.wait_for_consensus_path(path_type=ConsensusPathType.OPTIMISTIC_FAST,
+                                                  run_ops=lambda: self.send_kvs_sequentially(skvbc, op_count),
+                                                  threshold=op_count)
 
-    @unittest.skip("This is a transition covered in test_commit_path_transitions and is kept as a manual testing option.")
+    # @unittest.skip("This is a transition covered in test_commit_path_transitions and is kept as a manual testing option.")
```

Comment:
Did you mean to re-enable this test or you simply uncommented to test it?

Reply:
Uncommented to test it